### ISSUE TITLE
Honor newlines in Bios

### ISSUE THREAD

**Issue by joescii** [DevSpaceHuntsville/WebsiteAndApi#39]
Currently a bio cannot have any newlines. Hence this three-paragraph bio: 

![image](https://user-images.githubusercontent.com/882948/29689465-dfe4e60e-88e8-11e7-9759-4f99b27c4688.png)

Is jumbled up into one paragraph: 

![image](https://user-images.githubusercontent.com/882948/29689484-f9736186-88e8-11e7-94d6-0046095a7f87.png)

**Comment by freestylecoder** [comment]
This was addressed by merged PR #40.  Leaving the Issue open because we might want to switch to markdown later.

**Comment by ProfessorTom** [comment]
Now that we're several months away from this PR, do you want to go ahead and close it perhaps after creating an issue to support Markdown?

**Comment by freestylecoder** [comment]
No, I want to leave this here until such time as it's fixed.  If that gets fixed because someone added markdown support, then that would close this issue.

**Comment by ProfessorTom** [comment]
Perhaps the name of this issue should change to "Speaker Bios should support Markdown"

**Comment by freestylecoder** [comment]
No, because I’m not sure that is the best fix. It may be overkill for this problem 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
